lib/tool_mount_point/wpcc.rb

```diff
@@ -0,0 +1,6 @@
+module ToolMountPoint
+  class Wpcc < Base
+    EN_ID = 'wpcc'
```

Comment:
According to the [TP feature #8191](https://moneyadviceservice.tpondemand.com/entity/8191) the English and Welsh ID should be `workplace-pension-contribution-calculator` and `cyfrifiannell-cyfraniadau-pensiwn-gweithle`, respectively.

```
Tool will be hosted on the following link, which is currently being used by the existing WPCC (no redirect required) 

EN: https://www.moneyadviceservice.org.uk/en/tools/workplace-pension-contribution-calculator
CY: https://www.moneyadviceservice.org.uk/cy/tools/cyfrifiannell-cyfraniadau-pensiwn-gweithle
```